[2024-05-05T15:43:01.130+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-05T15:43:01.253+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: example_complex.get_entry_group_result manual__2024-05-05T15:42:35.204677+00:00 [queued]>
[2024-05-05T15:43:01.307+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: example_complex.get_entry_group_result manual__2024-05-05T15:42:35.204677+00:00 [queued]>
[2024-05-05T15:43:01.319+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 1
[2024-05-05T15:43:01.466+0000] {taskinstance.py:2327} INFO - Executing <Task(BashOperator): get_entry_group_result> on 2024-05-05 15:42:35.204677+00:00
[2024-05-05T15:43:01.544+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=191) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-05-05T15:43:01.546+0000] {standard_task_runner.py:63} INFO - Started process 206 to run task
[2024-05-05T15:43:01.550+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'example_complex', 'get_entry_group_result', 'manual__2024-05-05T15:42:35.204677+00:00', '--job-id', '8', '--raw', '--subdir', '/home/***/.local/lib/python3.12/site-packages/***/example_dags/example_complex.py', '--cfg-path', '/tmp/tmp__7e8uvx']
[2024-05-05T15:43:01.580+0000] {standard_task_runner.py:91} INFO - Job 8: Subtask get_entry_group_result
[2024-05-05T15:43:01.792+0000] {task_command.py:426} INFO - Running <TaskInstance: example_complex.get_entry_group_result manual__2024-05-05T15:42:35.204677+00:00 [running]> on host edcd6c8e3dad
[2024-05-05T15:43:02.114+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='example_complex' AIRFLOW_CTX_TASK_ID='get_entry_group_result' AIRFLOW_CTX_EXECUTION_DATE='2024-05-05T15:42:35.204677+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-05-05T15:42:35.204677+00:00'
[2024-05-05T15:43:02.116+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-05T15:43:02.117+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-05-05T15:43:02.118+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo get_entry_group_result']
[2024-05-05T15:43:02.175+0000] {subprocess.py:86} INFO - Output:
[2024-05-05T15:43:02.187+0000] {subprocess.py:93} INFO - get_entry_group_result
[2024-05-05T15:43:02.200+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-05-05T15:43:02.201+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-05T15:43:02.253+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=example_complex, task_id=get_entry_group_result, execution_date=20240505T154235, start_date=20240505T154301, end_date=20240505T154302
[2024-05-05T15:43:02.385+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-05T15:43:02.456+0000] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-05-05T15:43:02.476+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
